Sample_v3/Samples/CombineSimpleChat/CombineSimpleChatViewController.swift

```diff
@@ -0,0 +1,437 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Combine
+import StreamChatClient
+import UIKit
+
+///
+/// # CombineSimpleChatViewController
+///
+/// A `UITableViewController` subclass that displays and manages a channel.  It uses the `ChannelController`  class to make calls to the Stream Chat API and listens to
+/// events by conforming to `ChannelControllerDelegate`.
+///
+@available(iOS 13, *)
+final class CombineSimpleChatViewController: UITableViewController {
+    // MARK: - Properties
+    
+    ///
+    /// # channelController
+    ///
+    ///  The property below holds the `ChannelController` object.  It is used to make calls to the Stream Chat API and to listen to the events. After it is set,
+    ///  we need to start observing `ChannelController` event.
+    ///  While using Combine we should subscribe to `Publishers` events.
+    var channelController: ChannelController! {
+        didSet {
+            subscribeToCombinePublishers()
+        }
+    }
+    
+    // MARK: - Combine
+
+    private lazy var cancellables: Set<AnyCancellable> = []
+    
+    ///
+    /// # subscribeToCombinePublishers
+    ///
+    /// Here we bind `channelControllers` publishers so we can observe the changes.
+    private func subscribeToCombinePublishers() {
+        /// `ChannelController` will not trigger the `channelChangePublisher` on the initial channel set so
+        /// we can` prepend` our `channelChangePublisher` sequence with the initial channel manually.
+        let initialChannel = Just(channelController.channel)
+            .compactMap { $0 }
+            .map { EntityChange<Channel>.update($0) }
```

Comment:
I'm not sure this is a good idea actually. It's desired that the locally available channels are loaded without animation, only the changes are animated. Seem to me that the preferred solution is just to call `reloadData()` in `channelController { didSet { } }`?

Reply:
Seems like we mixed up here.
It's channel in "Chat" VC with messages.